From ecf2813e6eb9b6b4252d3bc283ab7e3f71af6341 Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Mon, 22 Apr 2013 17:04:28 -0400
Subject: [PATCH 1/7] Use control as mod key

---
 config.def.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 2d854a0..057ea84 100644
--- a/config.def.h
+++ b/config.def.h
@@ -93,7 +93,7 @@ static Mousekey mshortcuts[] = {
 };
 
 /* Internal keyboard shortcuts. */
-#define MODKEY Mod1Mask
+#define MODKEY ControlMask
 
 static Shortcut shortcuts[] = {
 	/* modifier		key		function	argument */
@@ -350,4 +350,3 @@ static Key key[] = {
 static uint selmasks[] = {
 	[SEL_RECTANGULAR] = Mod1Mask,
 };
-
-- 
1.8.2.3

