From fdca0f799052a90c51ef4bb5e7d924aeeff1008f Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Mon, 22 Apr 2013 17:04:42 -0400
Subject: [PATCH 2/6] Use 2 spaces for tabs

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index a30c73a..6c528a6 100644
--- a/config.def.h
+++ b/config.def.h
@@ -23,7 +23,7 @@ static unsigned int actionfps = 30;
 /* TERM value */
 static char termname[] = "st-256color";
 
-static unsigned int tabspaces = 8;
+static unsigned int tabspaces = 2;
 
 
 /* Terminal colors (16 first used in escape sequence) */
-- 
1.8.2.1

