From 1574bff5c4189eacb5c8b4dbc1d2233207189081 Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Mon, 22 Apr 2013 17:05:29 -0400
Subject: [PATCH 3/6] Use terminus font

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 6c528a6..499fb64 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=false:autohint=false";
+static char font[] = "-*-terminus-medium-r-*-*-10-*-*-*-*-*-*-*";
 static int borderpx = 2;
 static char shell[] = "/bin/sh";
 
-- 
1.8.2.1

