From 3690d12b339cd8d8447cf9cc0b5f631dd640d6e2 Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Mon, 22 Apr 2013 17:05:39 -0400
Subject: [PATCH 4/6] Use zsh

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index ac501e0..53905f7 100644
--- a/config.def.h
+++ b/config.def.h
@@ -7,7 +7,7 @@
  */
 static char font[] = "-*-terminus-medium-r-*-*-10-*-*-*-*-*-*-*";
 static int borderpx = 2;
-static char shell[] = "/bin/sh";
+static char shell[] = "/bin/zsh";
 
 /*
  * word delimiter string
-- 
1.8.2.2

