From 6f2c447393a99666481207935213472d57c09e7d Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Wed, 20 Feb 2013 22:17:51 -0500
Subject: [PATCH 2/3] Change to downloads directory before calling curl

Signed-off-by: Chirantan Ekbote <chirantan.ekbote@gmail.com>
---
 config.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.h b/config.h
index bbfb1ed..5d7a4dd 100644
--- a/config.h
+++ b/config.h
@@ -28,8 +28,8 @@ static Bool hidebackground  = FALSE;
 /* DOWNLOAD(URI, referer) */
 #define DOWNLOAD(d, r) { \
 	.v = (char *[]){ "/bin/sh", "-c", \
-		"st -e /bin/sh -c \"curl -J -O --user-agent '$1'" \
-		" --referer '$2' -b $3 -c $3 '$0';" \
+		"st -e /bin/sh -c \"cd ~/downloads && curl -J -O"\
+                " --user-agent '$1' --referer '$2' -b $3 -c $3 '$0';" \
 		" sleep 5;\"", \
 		d, useragent, r, cookiefile, NULL \
 	} \
-- 
1.8.1.3

