From 6af80754f9fae1aa97b9efe587b55bdfa1689d9b Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Thu, 7 Mar 2013 17:03:55 -0500
Subject: [PATCH 2/3] Change to downloads directory before calling curl

Signed-off-by: Chirantan Ekbote <chirantan.ekbote@gmail.com>
---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 533a149..0434a03 100644
--- a/config.def.h
+++ b/config.def.h
@@ -29,8 +29,8 @@ static Bool hidebackground  = FALSE;
 /* DOWNLOAD(URI, referer) */
 #define DOWNLOAD(d, r) { \
 	.v = (char *[]){ "/bin/sh", "-c", \
-		"st -e /bin/sh -c \"curl -J -O --user-agent '$1'" \
-		" --referer '$2' -b $3 -c $3 '$0';" \
+		"st -e /bin/sh -c \"cd ~/downloads && curl -J -O"\
+		" --user-agent '$1' --referer '$2' -b $3 -c $3 '$0';" \
 		" sleep 5;\"", \
 		d, useragent, r, cookiefile, NULL \
 	} \
-- 
1.8.1.5

