From addb8bc4c0d9bcac87b6a41b478455f864c90e0e Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Sat, 9 Mar 2013 14:58:00 -0500
Subject: [PATCH 3/6] Don't show the status bar on startup

Signed-off-by: Chirantan Ekbote <chirantan.ekbtoe@gmail.com>
---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 7bce89c..02739cd 100644
--- a/config.def.h
+++ b/config.def.h
@@ -10,7 +10,7 @@ static const char selbgcolor[]      = "#005577";
 static const char selfgcolor[]      = "#eeeeee";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
-static const Bool showbar           = True;     /* False means no bar */
+static const Bool showbar           = False;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
 
 /* tagging */
-- 
1.8.1.5

