From d162b92cc19889fb65da90bb1cb5d8dccbc69549 Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Wed, 27 Mar 2013 14:51:16 -0400
Subject: [PATCH 4/4] Make border color black, add volume control options

---
 config.def.h | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index 19498f0..2520b12 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@ static const char font[]            = "-*-terminus-medium-r-*-*-10-*-*-*-*-*-*-*
 static const char normbordercolor[] = "#444444";
 static const char normbgcolor[]     = "#222222";
 static const char normfgcolor[]     = "#bbbbbb";
-static const char selbordercolor[]  = "#005577";
+static const char selbordercolor[]  = "#000000";
 static const char selbgcolor[]      = "#005577";
 static const char selfgcolor[]      = "#eeeeee";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
@@ -52,6 +52,9 @@ static const Layout layouts[] = {
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
 static const char *termcmd[]  = { "st", "-f", font, NULL };
+static const char *mutecmd[] = { "amixer", "set", "Master", "toggle", NULL };
+static const char *raisevolumecmd[] = { "amixer", "set", "Master", "5%+", NULL };
+static const char *lowervolumecmd[] = { "amixer", "set", "Master", "5%-", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
@@ -78,6 +81,9 @@ static Key keys[] = {
 	{ MODKEY,                       XK_period, focusmon,       {.i = +1 } },
 	{ MODKEY|ShiftMask,             XK_comma,  tagmon,         {.i = -1 } },
 	{ MODKEY|ShiftMask,             XK_period, tagmon,         {.i = +1 } },
+	{ 0,                            0x1008ff12,spawn,          {.v = mutecmd } },
+	{ 0,                            0x1008ff11,spawn,          {.v = lowervolumecmd } },
+	{ 0,                            0x1008ff13,spawn,          {.v = raisevolumecmd } },
 	TAGKEYS(                        XK_1,                      0)
 	TAGKEYS(                        XK_2,                      1)
 	TAGKEYS(                        XK_3,                      2)
@@ -106,4 +112,3 @@ static Button buttons[] = {
 	{ ClkTagBar,            MODKEY,         Button1,        tag,            {0} },
 	{ ClkTagBar,            MODKEY,         Button3,        toggletag,      {0} },
 };
-
-- 
1.8.2.3

