From 89342f47b2566d5810c8cc58742c85905ca9d814 Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Sat, 9 Mar 2013 15:14:15 -0500
Subject: [PATCH 4/7] Use st with tmux as default terminal

Signed-off-by: Chirantan Ekbote <chirantan.ekbote@gmail.com>
---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 02739cd..a759085 100644
--- a/config.def.h
+++ b/config.def.h
@@ -51,7 +51,7 @@ static const Layout layouts[] = {
 
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char *termcmd[]  = { "st", "-e", "tmux", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
-- 
1.8.2

