From 31cdf75ec23bd3ae4d3afbe04992cfac1ef6eddf Mon Sep 17 00:00:00 2001
From: Chirantan Ekbote <chirantan.ekbote@gmail.com>
Date: Sat, 9 Mar 2013 15:37:17 -0500
Subject: [PATCH 6/6] Compile with icc

Signed-off-by: Chirantan Ekbote <chirantan.ekbote@gmail.com>
---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index 98e5bfa..4b4e74b 100644
--- a/config.mk
+++ b/config.mk
@@ -29,4 +29,4 @@ LDFLAGS  += -s ${LIBS}
 #LDFLAGS = ${LIBS}
 
 # compiler and linker
-CC = cc
+CC = icc
-- 
1.8.1.5

